extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -23,6 +23,26 @@ import {toggle} from '../../../src/style';
 
 
 class AmpStickyAd extends AMP.BaseElement {
+  /** @param {!AmpElement} element */
+  constructor(element) {
+    super(element);
+
+    /** @private {!Element} */
```

Comment:
wait. I am confused. Do we want to modify the input type of `#scheduleLayout()` to accept nullable value?  Or do we want to assign a fake value to satisfy the `#scheduleLayout()` input must not be null check?   for example `this.ad_ = element` (element is actually the container element) 
I am applying the later one now. I believe https://github.com/ampproject/amphtml/blob/master/extensions/amp-sticky-ad/0.1/amp-sticky-ad.js#L36 already does the check for me